parsers/BR.py

```diff
@@ -55,62 +63,51 @@ def get_data(session: Optional[Session]):
     return json_data
 
 
-def production_processor(json_data, zone_key: str) -> tuple:
+def production_processor(
+    json_data: dict, zone_key: str
+) -> Tuple[datetime, ProductionMix]:
     """Extracts data timestamp and sums regional data into totals by key."""
 
-    dt = arrow.get(json_data["Data"])
+    dt = arrow.get(json_data["Data"]).datetime
```

Comment:
This looks to be a propper ISO date string, we can probably replace arrow with datetime.fromisoformat() here.

Reply:
True that would be even better!